.github/workflows/scanner-mapping-update.yaml

```diff
@@ -3,40 +3,44 @@ name: Scanner mappings update
 on:
   schedule:
     - cron: "0 0 * * *"
+  push:
```

Comment:
From what I can tell, it's there now

```
curl -o mapping.zip https://definitions.stackrox.io/redhat-repository-mappings/mapping.zip
```

so just be sure to remove this prior to merge :)